x/genutil/migrations/v043/migrate.go

```diff
@@ -6,25 +6,25 @@ import (
 	v043bank "github.com/cosmos/cosmos-sdk/x/bank/migrations/v043"
 	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/cosmos/cosmos-sdk/x/genutil/types"
-	v040gov "github.com/cosmos/cosmos-sdk/x/gov/migrations/v040"
 	v043gov "github.com/cosmos/cosmos-sdk/x/gov/migrations/v043"
 	gov "github.com/cosmos/cosmos-sdk/x/gov/types"
+	oldGov "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
 // Migrate migrates exported state from v0.40 to a v0.43 genesis state.
 func Migrate(appState types.AppMap, clientCtx client.Context) types.AppMap {
```

Comment:
should the community pool token balance (distribution module account) be moved to the gov module account balance here? otherwise there would need to be an authz permission from the distribution module account to the gov module account and then community spend proposals would have to be submitted as authz transactions which would be a lot more complicated than just letting the gov module hold the community pool right? (also would need a change in the fee module so that community pool rewards are sent to the gov module in an ongoing basis)

Reply:
This is a good point. I will write up a follow up issue for this